TestExample/Support/AspireAppHostFixture.cs

```diff
@@ -0,0 +1,21 @@
+using Projects;
+
+namespace TestExample.Support;
+
+public class AspireAppHostFixture : IAsyncLifetime
+{
+    public DistributedApplication DistributedApplicationInstance { get; private set; } = null!;
+
+    public async Task InitializeAsync()
+    {
+        var appHost = await DistributedApplicationTestingBuilder.CreateAsync<WaitForDependencies_AppHost>();
+        DistributedApplicationInstance = await appHost.BuildAsync();
+        await DistributedApplicationInstance.StartAsync();
+    }
+    
+    public async Task DisposeAsync()
+    {
+        await DistributedApplicationInstance.StopAsync();
```

Comment:
```suggestion
```

Reply:
Cheers - added a commit to remove this, and return the Disposal ValueTask as a Task directly rather than awaiting within the method.